presto-tests/src/test/java/com/facebook/presto/tests/TestTpchDistributedStats.java

```diff
@@ -26,9 +29,110 @@ public TestTpchDistributedStats()
     {}
 
     @Test
-    void testEstimateForSimpleQuery()
+    void testTableScanStats()
     {
-        assertEquals(compareMetric("SELECT * FROM NATION").result(), MATCH);
+        assertEquals(compareMetric("SELECT * FROM nation").result(), MATCH);
+        assertEquals(compareMetric("SELECT * FROM part").result(), MATCH);
+        assertEquals(compareMetric("SELECT * FROM partsupp").result(), MATCH);
+        assertEquals(compareMetric("SELECT * FROM region").result(), MATCH);
+        assertEquals(compareMetric("SELECT * FROM supplier").result(), MATCH);
+        assertEquals(compareMetric("SELECT * FROM lineitem").result(), MATCH);
+        assertEquals(compareMetric("SELECT * FROM lineitem").result(), MATCH);
     }
 
+    @Test
+    void testFilter()
+    {
+        String query = "" +
+                "SELECT * " +
+                "FROM lineitem " +
+                "WHERE l_shipdate <= DATE '1998-12-01' - INTERVAL '90' DAY";
+        MetricComparison metricComparison = compareMetric(query);
+        assertEquals(metricComparison.result(), DIFFER);
+        assertEquals(metricComparison.getEstimate().get(), 30000, 1);
```

Comment:
The tolerance used is somewhat arbitrary and kinda small. I suspect it could be `0` equally well. This can mean two things:
1. we don't need to customize the tolerance and can live with the default behaviour of percentage tolerance of 10% (that's the behaviour of `result()` method). These are unit tests after all, and we control the entirety of TPCH connector.
2. we do need to customize the tolerance, but we don't know the actual tolerances yet. Then, the comments above apply.